config/500-controller.yaml

```diff
@@ -20,6 +20,7 @@ metadata:
   labels:
     contrib.eventing.knative.dev/release: devel
     control-plane: github-controller-manager
+    contrib.eventing.knative.dev/source-provider: github
```

Comment:
Is this something that we define in the spec somewhere?

I'd like to double check this with the Source WG lead, to make sure we have a consistent behavior for all our sources' metadata.

Can you take a look here, @n3wscott ? Thanks!

Reply:
@matzew @n3wscott Does it make more sense to use the label `sources.knative.dev/source-producer: github`?